pom.xml

```diff
@@ -57,7 +56,7 @@
         <as.managementPort>9990</as.managementPort>
         <browser>phantomjs</browser>
         <linkXRef>false</linkXRef>
-        <testsuite.config.location>${user.dir}/testsuite.properties</testsuite.config.location>
+        <testsuite.config.location>${user.dir}/org.jboss.hal.testsuite.properties</testsuite.config.location>
```

Comment:
Just one thing: is there some reason for `testsuite.properties` -> `org.jboss.hal.testsuite.properties`?

Reply:
Fixed, thank you!